.github/ISSUE_TEMPLATE/general-support.yml

```diff
@@ -0,0 +1,433 @@
+Skip to content
+Search or jump to…
+Pull requests
+Issues
+Codespaces
+Marketplace
+Explore
+ 
+@fajar-apri-alaska 
+AlaskaAirlines
+/
+auro-header
+Public
+Fork your own copy of AlaskaAirlines/auro-header
+Code
+Issues
+7
+Pull requests
+Actions
+Projects
+Security
+Insights
+Node 18+ updates #45
+ Merged
+blackfalcon merged 14 commits into main from dsande/node18update 3 days ago
++25,492 −16,077 
+ Conversation 1
+ Commits 14
+ Checks 2
+ Files changed 40
+ Merged
+Node 18+ updates
+#45
+Show file tree Hide file tree
+File filter 
```

Comment:
This whole file is incorrect? See reference https://github.com/AlaskaAirlines/auro-badge/pull/45/commits/a626ef293c05e6abaf4e222b3e3f40bd24ed7b15#diff-f98eee0578914954ee0aee3193f65adbf0f477f7ee2a3edf11570d25777bed08

Reply:
did I just do this? Let me check, sorry.